get-test-results.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+# Default values
+timeout=600
+resultsDir="/app/allure-results"
+resultFilePrefixes=("mp-authority" "mp-supplier" "pb")
```

Comment:
nightmare, we'll need to keep adding to this as we add new test packs. Can we not scoop this info somehow?